src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/CursorIT.java

```diff
@@ -0,0 +1,373 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.esintgtest;
+
+import com.amazon.opendistroforelasticsearch.sql.utils.StringUtils;
+import org.elasticsearch.client.Request;
+import org.elasticsearch.client.Response;
+import org.elasticsearch.client.ResponseException;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static com.amazon.opendistroforelasticsearch.sql.esintgtest.TestUtils.getResponseBody;
+import static com.amazon.opendistroforelasticsearch.sql.esintgtest.TestsConstants.TEST_INDEX_ACCOUNT;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.containsString;
+
+public class CursorIT extends SQLIntegTestCase {
```

Comment:
I assume cursor can work with PreparedStatement. If so, could you check if our cursor implementation can work with prepared statement request and add test case here?

Reply:
Sure, will confirm and add a test case as well.

Reply:
Thanks!

Reply:
Done.